clevertap-core/src/main/java/com/clevertap/android/sdk/cryption/CryptUtils.kt

```diff
@@ -19,7 +19,7 @@ import org.json.JSONObject
 /**
  * This class is a utils class, mainly used to handle migration when encryption fails or encryption level is changed
  */
-object CryptUtils {
+internal object CryptUtils {
```

Comment:
Made internal because I changed the DBAdapter to be internal instead of RestrictTo(Library) annotation and this class exposed DBAdapter. I think it should also probably be internal so I changed it as well.